systemtests/volsupervisor_test.go

```diff
@@ -73,19 +48,19 @@ func (s *systemtestSuite) TestVolsupervisorSnapshotSchedule(c *C) {
 
 	c.Assert(s.createVolume("mon0", fqVolume("policy1", volName), map[string]string{"unlocked": "true"}), IsNil)
 
-	time.Sleep(4 * time.Second)
+	time.Sleep(10 * time.Second)
 
 	out, err := s.vagrant.GetNode("mon0").RunCommandWithOutput("sudo rbd snap ls policy1." + volName)
 	c.Assert(err, IsNil)
-	c.Assert(len(strings.Split(out, "\n")) > 2, Equals, true)
+	c.Assert(len(strings.Split(strings.TrimSpace(out), "\n"))-1 > 2, Equals, true)
```

Comment:
I have limited time. I can look into which ones are not accommodating 
the header, but I suspect all the ones that are (were) failing were 
related.

What we should really do is output these in the ceph json format and 
parse it that way, but alas, again, time.

On 22 Aug 2016, at 14:45, Yuva Shankar wrote:

>>
>>  	out, err := s.vagrant.GetNode("mon0").RunCommandWithOutput("sudo 
>> rbd snap ls policy1." + volName)
>>  	c.Assert(err, IsNil)
>> -	c.Assert(len(strings.Split(out, "\n")) > 2, Equals, true)
>> +	c.Assert(len(strings.Split(strings.TrimSpace(out), "\n"))-1 > 2, 
>> Equals, true)
>
> Any specific reason why is it not been considered on other test cases 
> ?
>
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/435/files/f2e5743d552dd2415c24d3efcdd4f70cc7759f41#r75765205